components/gcp/dataproc/create_cluster/README.md

```diff
@@ -97,38 +102,29 @@ def dataproc_create_cluster_pipeline(
     project_id = PROJECT_ID, 
     region = 'us-central1', 
     name='', 
-    name_prefix='',
+    name_prefix='', 
+    job_name_prefix='', 
     initialization_actions='', 
     config_bucket='', 
     image_version='', 
     cluster='', 
     wait_interval='30'
 ):
-    dataproc_create_cluster_op(
-        project_id=project_id, 
-        region=region, 
-        name=name, 
-        name_prefix=name_prefix, 
-        initialization_actions=initialization_actions, 
-        config_bucket=config_bucket, 
-        image_version=image_version, 
-        cluster=cluster, 
-        wait_interval=wait_interval).apply(gcp.use_gcp_secret('user-gcp-sa'))
+    dataproc_create_cluster_op(project_id, region, name, name_prefix, job_name_prefix, initialization_actions, 
+       config_bucket, image_version, cluster, wait_interval).apply(gcp.use_gcp_secret('user-gcp-sa'))
 ```
```

Comment:
General convention in other files it to split the variables on their own individual lines

Reply:
I leave it up to @hongye-sun to decide, I don't have any preference on this.